cloud_providers/pharos-in-tree.md

```diff
@@ -0,0 +1,81 @@
+# Pharos "in-tree" cloud providers
+
+Support for these cloud prviders is built-in with Pharos and they can be configured as normal Kubernetes supported providers:
```

Comment:
Are we missing some content here?

Reply:
No, this is mainly a list with links for specific providers